recipes/web.py/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "web.py" %}
+{% set version = "0.40.dev0" %}
+{% set sha256 = "d464fa8d7cdd90c3a8a163b1b5e8e55b856cca4d13a9938e3a7920a3090b69a3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
It looks like it's required:
https://travis-ci.org/conda-forge/staged-recipes/builds/260083186#L234

`ImportError: No module named setuptools`